src/main/java/org/apache/commons/collections4/list/CursorableLinkedList.java

```diff
@@ -279,7 +279,7 @@ protected void registerCursor(final Cursor<E> cursor) {
     }
 
     /**
-     * Deregisters a cursor from the list to be notified of changes.
+     * Deregister a cursor from the list to be notified of changes.
```

Comment:
garydgregory on 17 Apr Member
If a method starts with "make...", it usually reads better to start the Javadoc with "Makes..." IMO, It's confusing to have the Javadoc be different, otherwise you could say that all method that start with "set..." also "forces" a value.

Reply:
yes you are right.
will revert it.